ChaosMod/Effects/db/Misc/MiscLowPitch.cpp

```diff
@@ -0,0 +1,30 @@
+#include <stdafx.h>
+
+#include "Memory/Hooks/AudioPitchHook.h"
+
+//Effect by ProfessorBiddle, but the code is pretty much copied and pasted
+
+static int m_targetPitch;
+
+static void OnStart()
+{
+	m_targetPitch = g_random.GetRandomInt(-900, -300);
+
+	SET_AUDIO_FLAG("AllowScriptedSpeechInSlowMo", true);
+	SET_AUDIO_FLAG("AllowAmbientSpeechInSlowMo", true);
```

Comment:
No need to set / unset those flags as the actual game speed isn't being modified